open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/MenuShortCutsUI.java

```diff
@@ -0,0 +1,100 @@
+package io.opensphere.core.control.keybinding;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+import javafx.scene.layout.HBox;
+
+/**
+ * Creates a table/legend of the current Opensphere shortcut keys which pertain
+ * to "File","Edit","Tools" Menus.
+ */
+public class MenuShortCutsUI extends HBox
+{
+    /**
+     * Creates the region containing the UI.
+     * 
+     * @param width the horizontal size desired.
+     * @param height the vertical size desired.
+     */
+    public MenuShortCutsUI(int width, int height)
+    {
+        setMinSize(width, height);
+        setStyle("-fx-background-color : derive(-fx-base, 18%)");
+        getChildren().add(createTableView(width, height));
+    }
+
+    /**
+     * Creates a JavaFX TableView for the "Menu ShortCuts" tab.
+     * 
+     * @param width the int to specify the total table width.
+     * @param height the int to specify the total table height.
+     * @return theTable a JavaFX TableView containing the Menu Shortcuts.
```

Comment:
No need to give the name of the return variable

Reply:
updated